test/unit/resources/org/folio/client/jira/JiraClientTest_testIssueAddComment.groovy

```diff
@@ -3,7 +3,7 @@ package org.folio.version
 import org.folio.client.jira.JiraClient
 
 def execute() {
-    def client = new JiraClient(this, "https://issues.folio.org", "dummy", "dummy")
```

Comment:
legacy URL was still in use. madness :)

Reply:
It will be rebased in the following PRs.